include/irr/asset/ICPUMeshBuffer.h

```diff
@@ -84,11 +84,37 @@ class ICPUMeshBuffer : public IMeshBuffer<ICPUBuffer, ICPUDescriptorSet, ICPURen
     {
         return &m_indexBufferBinding;
     }	
+	[[nodiscard]] inline bool setVertexBufferBinding(SBufferBinding&& bufferBinding, uint32_t bindingIndex, uint32_t stride, E_VERTEX_INPUT_RATE inputRate = E_VERTEX_INPUT_RATE::EVIR_PER_VERTEX)
+	{
+		if (stride >= 2048ull || bindingIndex >= MAX_ATTR_BUF_BINDING_COUNT)
+			return false;
+
+		m_vertexBufferBindings[bindingIndex].buffer = std::move(bufferBinding.buffer);
+		m_vertexBufferBindings[bindingIndex].offset = bufferBinding.offset;
+
+		auto& binding(m_pipeline->getVertexInputParams().bindings[bindingIndex]);
```

Comment:
I'd like to set stride and input rate separately from the buffer bindning (3rd function please)